pkgdb2client/admin.py

```diff
@@ -391,15 +398,26 @@ def main():
     elif arg.verbose:
         LOG.setLevel(logging.INFO)
 
-    if arg.test:
-        global PKGDBCLIENT
-        print "Testing environment"
+    global PKGDBCLIENT
+    if arg.pkgdburl:
+        print "Querying pkgdb at: %s" % arg.pkgdburl
```

Comment:
In a way I agree, on the other side this is such an important change (results may/will vary) that I wonder if we shouldn't be explicit about it.

What do you think?